### PR TITLE
Electrical thread try catch

### PR DESCRIPTION
**Pull request by ThatDan123**
Might as well add it here too

-Also removed some unused atmos stuff

### CODE REVIEW

File: UnityProject/Assets/Scripts/Systems/Electricity/Electrical processes/ElectricalManager.cs [hunk 1]
```diff
@@ -59,7 +59,15 @@ private void UpdateMe()
...
 					lock (ElectricalLock)
 					{
-						electricalSync.PowerNetworkUpdate();
+						try
+						{
+							electricalSync.PowerNetworkUpdate();
+						}
+						catch (Exception e)
+						{
+							Logger.LogError($"Electrical MainThreadProcess Error! \n{e}", Category.Electrical);
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Systems/Electricity/Electrical processes/ElectricalManager.cs]: include the trace

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Systems/Electricity/Electrical processes/ElectricalManager.cs]: apparently you can just put `e` or `e.ToString()` and it'll put the whole message with stacktrace

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Systems/Electricity/Electrical processes/ElectricalManager.cs]: looked into the unity code for it and gagged, so ive made an extension instead

**Review by corp-0 — APPROVED** [review]

**Comment by Bod9001** [comment]
Might get a bit Spamy with errors but Should catch those Weird instances where it is not initialised properly

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**